### PR TITLE
[IRRC-68 & 70] update multiple students' status and english proficiency, ensure that updated data is reflected in student profile

### PR DESCRIPTION
**Pull request by nondescryptid**

### CODE REVIEW

**Comment by nondescryptid** [comment]
Are there plans to add new statuses? If not, would it be ok to hard code the statusID to update the backend with (StatusID: 2 for Unmatched) instead of using StatusString (which involves more calls to the API)?

**Comment by thchia** [comment]
@szktm we can’t assume those IDs will be stable. You can see yourself if you run the seeds locally a few times that the IDs can change.

**Comment by nondescryptid** [comment]
Note: May be affected by #54 (will update this after reviewing that PR)

File: backend/src/routes/screening.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const express = require("express");
+const router = express.Router();
+// const debug = require('debug')('app:students')
+const students = require("../helpers/students");
+const statuses = require("../helpers/statuses.js")
+const { UniqueViolationError } = require("objection");
+const { NotFoundError } = require("objection");
+const { response } = require("express");
+const { restart } = require("nodemon");
```
Comment (thchia) [inline backend/src/routes/screening.js]: These 2 are accidental `require`s.

File: frontend/src/store/screening.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+export const screeningActions = {
+    async patchScreeningStudents({commit}, data){
+        const patchScreening = {
+            method: "POST",
+            headers: {
+              'Accept': 'application/json',
+              'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(data)
+          }
```
Comment (thchia) [inline frontend/src/store/screening.js:14]: I saw this after #60 and #61, and how good is it that we have implemented this the same way!

**Review by thchia — APPROVED** [review]
No issues with this, just some code that can be cleaned up. Note I didn't test, just looked at the code.

**Review by dzldebby — APPROVED** [review]
I can change english proficiency of multiple students, and was also able to see them reflected correctly on the all students table as well as the individual student profile pages. Awesome 👍 👍 👍  LGTM :)

**Pull request merged**